samples/charts/data-chart/series-marker-template/Pages/DataChartSeriesMarkerTemplate.razor

```diff
@@ -0,0 +1,56 @@
+﻿@page "/DataChartSeriesMarkerTemplate"
+
+@using IgniteUI.Blazor.Controls
+@inject IIgniteUIBlazor IgniteUIBlazor;
+@inject IJSRuntime JSRuntime;
+
+<div class="igContainer">
+    <div class="igComponent">
+        @if (DataSource != null)
+        {
+            <DataChart Height="100%" Width="100%">
+                <CategoryXAxis Name="xAxis" DataSource="DataSource" Label="Label" Interval="1"  />
+                <NumericYAxis Name="yAxis" MinimumValue="20" MaximumValue="120" />
+                
+                <ColumnSeries XAxisName="xAxis" YAxisName="yAxis" DataSource="DataSource"
+                              ValueMemberPath="Value" MarkerTemplateScript="onDataChartSeriesMarkerTemplate" />
```

Comment:
add a comment with a note saying that the onDataChartSeriesMarkerTemplate function is implemented in DataChartSeriesMarkerTemplate.js file